fleet/lib/kube-prometheus-stack/values.yaml

```diff
@@ -121,6 +119,10 @@ coreDns:
   serviceMonitor:
     additionalLabels:
       lsst.io/monitor: "true"
+kubeDns:
```

Comment:
This was removed because we don't use kubeDns. https://github.com/lsst-it/k8s-cookbook/pull/445